pytorch_lightning/loggers/wandb.py

```diff
@@ -27,16 +26,14 @@
 from pytorch_lightning.callbacks.model_checkpoint import ModelCheckpoint
 from pytorch_lightning.loggers.base import LightningLoggerBase, rank_zero_experiment
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.imports import _compare_version, _module_available
+from pytorch_lightning.utilities.imports import _WANDB_GREATER_EQUAL_0_10_22, _WANDB_GREATER_EQUAL_0_12_10
 from pytorch_lightning.utilities.logger import _add_prefix, _convert_params, _flatten_dict, _sanitize_callable_params
 from pytorch_lightning.utilities.rank_zero import rank_zero_only, rank_zero_warn
 
-_WANDB_AVAILABLE = _module_available("wandb")
-_WANDB_GREATER_EQUAL_0_10_22 = _compare_version("wandb", operator.ge, "0.10.22")
-
 try:
-    import wandb
     from wandb.wandb_run import Run
+
+    import wandb
```

Comment:
```suggestion
    import wandb
    from wandb.wandb_run import Run
```
this will solve the pre-commit.
will look into monkeypatch tomorrow. Unfortunately can't commit to your PR.

Reply:
Strange… `isort` wants it in the other order for me…

Looks like the `wandb` fork is private, probably why you can't commit there. I could PR through my own fork.

Reply:
could it be that you are running a different version of isort?

Reply:
I checked that but was the same...
Could be that wandb cannot be installed or I have some local settings messing this up.